src/components/Icons/icons/IconCross.tsx

```diff
@@ -0,0 +1,17 @@
+import type { FC } from 'react';
```

Comment:
Nice - thanks for adding these! Just FYI we're about to add a script which automatically creates React components from all Figma icons 🚀